src/App.jsx

```diff
@@ -11,6 +12,8 @@ function App() {
         <Routes>
           <Route path="/" element={<Home />} />
           <Route path="/login" element={<Login />} />
+          {/* Rota temporaria, ate que rota da tabela de voluntarios nao estiver pronta */}
+          <Route path="/:id" element={<VoluteerDetails />} />
```

Comment:
A rota final vai ser `/voluntarios/:id`? Qual é o problema de deixar ela desse jeito já?

Também tem esse erro de digitação (também no import) - **~Voluteer~ -> Volunteer**
```suggestion
          {/* Rota temporaria, ate que rota da tabela de voluntarios nao estiver pronta */}
          <Route path="/:id" element={<VolunteerDetails />} />
```

Reply:
Sim @lucas-lm